app/models/checkout.rb

```diff
@@ -22,6 +18,14 @@ def status
     fields['status']
   end
 
+  def due_date
+    fields['dueDate'] && Time.zone.parse(fields['dueDate'])
+  end
+
+  def recall_due_date
+    fields['recallDueDate'] && Time.zone.parse(fields['recallDueDate'])
+  end
+
   def recalled_date
     Time.zone.parse(fields['recalledDate']) if fields['recalledDate']
   end
```

Comment:
Just want to point out that we aren't using this method anywhere except in `recalled?`. Seems like a good candidate to demote to `private`. Unless we want to display this information.

Reply:
It is used in the helper https://github.com/psu-libraries/myaccount/blob/cd4d4ac08f3ac59ac58aafd4f81e32902b87f6d9/app/helpers/checkouts_helper.rb#L21

Reply:
Ah, I see. Don't know how I missed that. Sorry.